crates/sound/src/audio_backends.rs

```diff
@@ -46,17 +52,23 @@ pub trait AudioBackend {
 }
 
 pub fn alloc_audio_backend(
-    name: String,
+    backend: BackendType,
     streams: Arc<RwLock<Vec<Stream>>>,
 ) -> Result<Box<dyn AudioBackend + Send + Sync>> {
-    log::trace!("allocating audio backend {}", name);
-    match name.as_str() {
+    use BackendType::*;
```

Comment:
I'd suggest avoiding this, since `cargo build --no-default-features ` is complaining about unused import.

What about this?
```patch
diff --git a/crates/sound/src/audio_backends.rs b/crates/sound/src/audio_backends.rs
index bd13c37..0d62591 100644
--- a/crates/sound/src/audio_backends.rs
+++ b/crates/sound/src/audio_backends.rs
@@ -55,15 +49,14 @@ pub fn alloc_audio_backend(
     backend: BackendType,
     streams: Arc<RwLock<Vec<Stream>>>,
 ) -> Result<Box<dyn AudioBackend + Send + Sync>> {
-    use BackendType::*;
     log::trace!("allocating audio backend {:?}", backend);
     match backend {
         #[cfg(feature = "null-backend")]
-        Null => Ok(Box::new(NullBackend::new(streams))),
+        BackendType::Null => Ok(Box::new(NullBackend::new(streams))),
         #[cfg(feature = "pw-backend")]
-        Pipewire => Ok(Box::new(PwBackend::new(streams))),
+        BackendType::Pipewire => Ok(Box::new(PwBackend::new(streams))),
         #[cfg(feature = "alsa-backend")]
-        Alsa => Ok(Box::new(AlsaBackend::new(streams))),
+        BackendType::Alsa => Ok(Box::new(AlsaBackend::new(streams))),
         #[cfg(not(all(
             feature = "null-backend",
             feature = "pw-backend",
```

Reply:
Yes, I could make the import optional too, but only three branches benefit from it, so it makes more sense to remove it as in your suggestion.

Anyway, does it make sense to compile without features? Maybe the Null backend should be always available.
In any case, it is out of the scope of this PR, so I will just follow your suggestion :)

Reply:
Yes, maybe it doesn't make sense, and we should have the null backend always compiled. It's just that we have a feature to disable the null backend when not needed, and so I don't know how to make sure that there is always a compiled backend.

Maybe playing with Cargo.toml can be done, but I don't know if it's worth it ;-)

Reply:
Without features it also warns about unused variable with
```rust
streams: Arc<RwLock<Vec<Stream>>>,
```
Should I change it to `_streams` (in a separate commit)?